en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -52,78 +56,81 @@ The `Statistics` REST API opens on a separate page.
 
 ### Step 1.3: Design the REST API
 
-Let's design the REST API as follows:
+To design the REST API involves specifying how the REST API should function by adding and configuring the required connectors and statements. You can do this by editing the low-code diagram of the REST API or by editing its code. 
```

Comment:
"To design the REST API involves specifying..." doesn't seem correct. Might have to rephrase.

Reply:
Rephrased